<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1443</width>
    <height>1026</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolBar
{
	spacing:5px;
	border: none;
}

QToolButton
{
	margin: 2px 2px 2px 2px;
}

QLabel#itemNameLabel
{
	font: 16px;
}

QLabel#itemCodeLabel
{
	color: #999999;
}

QLabel#currentValueLabel
{
	font: bold 16px;
	color: #990000;
}

QGroupBox
{
	border: none;
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="fundTabWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>871</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="fundNetvalueWidget">
     <attribute name="title">
      <string>净值</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QFrame" name="nvFrame">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>9</number>
         </property>
         <item>
          <widget class="QLabel" name="fund_name_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fund_code_label">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="anv_name_label">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>单位净值</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="anv_value_label">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="anv_growthrate_label">
           <property name="text">
            <string>0.00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cnv_name_label">
           <property name="text">
            <string>累计净值</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cnv_value_label">
           <property name="text">
            <string>0.00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nv_lastest_date_label">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="fundNetValuePeriodGroupBox">
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="periodRadioButton1">
           <property name="text">
            <string>半年</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="periodRadioButton2">
           <property name="text">
            <string>一年</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="periodRadioButton3">
           <property name="text">
            <string>三年</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="periodRadioButton4">
           <property name="text">
            <string>五年</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="periodRadioButton5">
           <property name="text">
            <string>成立来</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="anvChartWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="cnvChartWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="fundHoldingWidget">
     <attribute name="title">
      <string>持仓</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>88</width>
           <height>800</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>800</height>
          </size>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="holdRatioWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="holdShareWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="holdBondWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="holdIndustryWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="fundScaleWidget">
     <attribute name="title">
      <string>规模</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="fundBasicWidget">
     <attribute name="title">
      <string>资料</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="fundBasicTableWidget">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="leftListWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>274</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="text">
        <string>所选基金列表</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="fundListWidget">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1443</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSync"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSetup"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作(O)</string>
    </property>
    <addaction name="actionTradeFund"/>
    <addaction name="separator"/>
    <addaction name="actionAnalyzeIndex"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="actionLogout"/>
   <addaction name="actionTradeFund"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionLogin">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ContactNew"/>
   </property>
   <property name="text">
    <string>登录(&amp;L)...</string>
   </property>
   <property name="iconText">
    <string>登录</string>
   </property>
   <property name="toolTip">
    <string>登录应用</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemShutdown"/>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
   <property name="iconText">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出当前应用程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>关于(&amp;A)...</string>
   </property>
   <property name="toolTip">
    <string>关于...</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::UserAvailable"/>
   </property>
   <property name="text">
    <string>注销(&amp;O)</string>
   </property>
   <property name="iconText">
    <string>注销</string>
   </property>
   <property name="toolTip">
    <string>注销账户</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaylistRepeat"/>
   </property>
   <property name="text">
    <string>数据更新(&amp;R)...</string>
   </property>
   <property name="iconText">
    <string>数据更新</string>
   </property>
   <property name="toolTip">
    <string>数据更新...</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
   <property name="text">
    <string>设置(&amp;S)</string>
   </property>
   <property name="iconText">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
  </action>
  <action name="actionAnalyzeIndex">
   <property name="text">
    <string>指数分析(&amp;I)...</string>
   </property>
   <property name="iconText">
    <string>指数分析</string>
   </property>
   <property name="toolTip">
    <string>指数分析(I)...</string>
   </property>
  </action>
  <action name="actionTradeFund">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaylistShuffle"/>
   </property>
   <property name="text">
    <string>基金交易(O)...</string>
   </property>
   <property name="iconText">
    <string>基金交易</string>
   </property>
   <property name="toolTip">
    <string>基金交易</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionLogin</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionLoginTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionExitTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLogout</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionLogoutTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSync</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionSyncTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSetup</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionSetupTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fundListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>mainWindow</receiver>
   <slot>onCurrentRowChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>955</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fundBasicTableWidget</sender>
   <signal>cellEntered(int,int)</signal>
   <receiver>mainWindow</receiver>
   <slot>onFundBasicTableWidgetCellEntered(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>863</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodRadioButton3</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>onFundNetValuePeriodChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1052</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>813</x>
     <y>765</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodRadioButton2</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>onFundNetValuePeriodChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>990</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>825</x>
     <y>824</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodRadioButton5</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>onFundNetValuePeriodChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1188</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>902</x>
     <y>717</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodRadioButton4</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>onFundNetValuePeriodChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1114</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>874</x>
     <y>737</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodRadioButton1</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>onFundNetValuePeriodChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>928</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>1137</x>
     <y>869</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAnalyzeIndex</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionAnalyzeIndexTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTradeFund</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionTradeFundTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onActionLoginTriggered()</slot>
  <slot>onActionLogoutTriggered()</slot>
  <slot>onActionExitTriggered()</slot>
  <slot>onTreeViewItemSelected(QModelIndex)</slot>
  <slot>onRefreshButtonClicked()</slot>
  <slot>onActionSyncTriggered()</slot>
  <slot>onActionListSecTriggered()</slot>
  <slot>onActionSetupTriggered()</slot>
  <slot>onCurrentRowChanged(int)</slot>
  <slot>onFundBasicTableWidgetCellEntered(int,int)</slot>
  <slot>onFundNetValuePeriodChanged()</slot>
  <slot>onActionAnalyzeIndexTriggered()</slot>
  <slot>onActionTradeFundTriggered()</slot>
 </slots>
</ui>
